packages/flutter_tools/lib/src/template.dart

```diff
@@ -328,6 +328,20 @@ class Template {
           context['androidIdentifier'] = _escapeKotlinKeywords(androidIdentifier);
         }
 
+        final bool web = (context['web'] as bool?) ?? false;
+
+        if (web) {
+          // The index.html and manifest.json outputs already have quotes in the template,
+          // so extra quotes are not needed.
```

Comment:
Should we have this comment should explain where the extra quotes came from?

Reply:
Since the 'extra' quotes have been eliminated from `createTemplateContext()`, I've adjusted the comment to mention why we add quotes instead.